app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -123,6 +123,33 @@ def index_field_link(options)
       safe_join(links, ", ")
     end
 
+    # A Blacklight helper_method
+    #
+    # @example
+    #   link_to_each_facet_field({ value: "Imaging > Object Photography", config: { helper_facet: :document_types_sim }})
+    #   ```html
+    #   <a href=\"/catalog?f%5Bdocument_types_sim%5D%5B%5D=Imaging\">Imaging</a> &gt; <a href=\"/catalog?f%5Bdocument_types_sim%5D%5B%5D=Object+Photography\">Object Photography</a>
+    #   ```
+    #
+    # @param options [Hash] from blacklight invocation of helper_method
+    # @option options [Array<#strip>] :value
+    # @option options [Hash>] :config Example: { separator: '>', helper_facet: :document_types_sim, output_separator: '::' }
+    # @return the html_safe facet links separated by the given separator (default: ' > ') to indicate facet hierarchy
+    #
+    # @raise KeyError when the options are note properly configured
+    def link_to_each_facet_field(options)
+      config = options.fetch(:config)
+      separator = config.fetch(:separator, '>')
+      output_separator = config.fetch(:output_separator, separator).to_s.strip
```

Comment:
Spaces get stripped from the option by default and have to be explicitly added back in. I think the dominant use case has spaces, not having any would severely hinder readability, be a usability problem. I made them the default, with a config that can override this. Will that work?